examples/plotting/histogram.py

```diff
@@ -16,16 +19,16 @@
 def make_plot(title, hist, edges, x, pdf, cdf):
     p = figure(title=title, tools='', background_fill_color="#fafafa")
     p.quad(top=hist, bottom=0, left=edges[:-1], right=edges[1:],
-           fill_color="navy", line_color="white", alpha=0.5)
-    p.line(x, pdf, line_color="#ff8888", line_width=4, alpha=0.7, legend_label="PDF")
-    p.line(x, cdf, line_color="orange", line_width=2, alpha=0.7, legend_label="CDF")
+           fill_color="#004488", line_color="#ffffff", alpha=0.5) #updated basic fill color from low contrast pallette(navy) to HighContrast (#004488)
+    p.line(x, pdf, line_color="#BB5566", line_width=4, alpha=0.7, legend_label="PDF") #updated line color from low contrast pallette(#ff8888) to HighContrast (#BB5566)
+    p.line(x, cdf, line_color="#DDAA33", line_width=2, alpha=0.7, legend_label="CDF") #updated basic line color from low contrast pallette(orange) to HighContrast (#DDAA33)
```

Comment:
Similar as above, these comments are a distraction for *users* trying to learn Bokeh (they have no context for the file history or why or when updates might have been made, and it does not inform how they will use the API themselves)
```suggestion
           fill_color="#004488", line_color="#ffffff", alpha=0.5)
    p.line(x, pdf, line_color="#BB5566", line_width=4, alpha=0.7, legend_label="PDF")
    p.line(x, cdf, line_color="#DDAA33", line_width=2, alpha=0.7, legend_label="CDF")
```